### DISCUSSION TITLE
Adding a new language.

### DISCUSSION THREAD

**Discussion by p6laris** [jellyfin/jellyfin#10901] [Q&A]
I would like to contribute by translating to Kurdish language, I don't see any option for kurdish in Weblate though!

**Comment by pedrojreis** [comment]
Hello @p6laris .

To do that you have to login into Weblate, go to the project (let's say Jellyfin) languages and scroll all the way to the bottom

![image](https://github.com/jellyfin/jellyfin/assets/28795057/5cabced7-e145-4eb9-a181-813134c5a2d9)

Pick your language

![image](https://github.com/jellyfin/jellyfin/assets/28795057/424d7773-b8d0-4f11-b723-670589c8c882)

And then start translating :)
